mne/tests/test_utils.py

```diff
@@ -414,6 +415,27 @@ def test_check_type_picks():
     assert_raises(ValueError, _check_type_picks, picks)
 
 
+def test_compute_corr():
+    """Test Anscombe's Quartett
+    """
+    x = np.array([10, 8, 13, 9, 11, 14, 6, 4, 12, 7, 5])
+    y = np.array([[8.04, 6.95, 7.58, 8.81, 8.33, 9.96,
+                   7.24, 4.26, 10.84, 4.82, 5.68],
+                  [9.14, 8.14, 8.74, 8.77, 9.26, 8.10,
+                   6.13, 3.10, 9.13, 7.26, 4.74],
+                  [7.46, 6.77, 12.74, 7.11, 7.81, 8.84,
+                   6.08, 5.39, 8.15, 6.42, 5.73],
+                  [8, 8, 8, 8, 8, 8, 8, 19, 8, 8, 8],
+                  [6.58, 5.76, 7.71, 8.84, 8.47, 7.04,
+                   5.25, 12.50, 5.56, 7.91, 6.89]])
+
+    r = compute_corr(x, y.T)
+    r2 = np.array([np.corrcoef(x, y[i])[0, 1]
+                   for i in range(len(y))])
+    assert_allclose(r, r2, atol=1e-3)
```

Comment:
why only an atol of 1e-3? I would expect a much better numerical accuracy for this